MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,71 +2,170 @@ import UIKit
+            correctAnswer: true),
+        QuizQuestion(
+            image: "Old",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false),
+        QuizQuestion(
+            image: "The Ice Age Adventures of Buck Wild",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false),
+        QuizQuestion(
+            image: "Tesla",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false),
+        QuizQuestion(
+            image: "Vivarium",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false)
+    ]
+    
+    @IBOutlet private weak var yesButton: UIButton!
+    @IBOutlet private weak var noButton: UIButton!
+    @IBOutlet private weak var questionTitleLabel: UILabel!
+    @IBOutlet private weak var questionLabel: UILabel!
+    @IBOutlet private weak var counterLabel: UILabel!
+    @IBOutlet private weak var imageView: UIImageView!
 
- Картинка: The Green Knight
- Настоящий рейтинг: 6,6
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
+    override func viewDidLoad() {
+        super.viewDidLoad()
 
+        yesButton.titleLabel!.font = UIFont(name: "YSDisplay-Medium", size: 20)
+        noButton.titleLabel!.font = UIFont(name: "YSDisplay-Medium", size: 20)
+        questionTitleLabel.font = UIFont(name: "YSDisplay-Medium", size: 20)
+        questionLabel.font = UIFont(name: "YSDisplay-Bold", size: 23)
+        counterLabel.font = UIFont(name: "YSDisplay-Medium", size: 20)
+        imageView.layer.masksToBounds = true
+        
+        let viewModel = convert(model: questions[currentQuestionIndex])
+        show(quiz: viewModel)
+    }
+    
+    @IBAction private func yesButtonClicked(_ sender: UIButton) {
+        let answer = true
+        let result = answer == questions[currentQuestionIndex].correctAnswer ? true : false
+        showAnswerResult(isCorrect: result)
+    }
 
- Картинка: Old
- Настоящий рейтинг: 5,8
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: НЕТ
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        let answer = false
+        let result = answer == questions[currentQuestionIndex].correctAnswer ? true : false
+        showAnswerResult(isCorrect: result)
+    }
 
+    private func show(quiz step: QuizStepViewModel) {
+        questionLabel.text = step.question
+        counterLabel.text = step.questionNumber
+        imageView.image = step.image
+        imageView.layer.borderWidth = 0
+    }
 
- Картинка: The Ice Age Adventures of Buck Wild
- Настоящий рейтинг: 4,3
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: НЕТ
+    private func show(quiz result: QuizResultsViewModel) {
+        let alert = UIAlertController(
+            title: result.title,
+            message: result.text,
+            preferredStyle: .alert)
+        let action = UIAlertAction(title: result.buttonText, style: .default) { [self]_ in
```

Comment:
Про слабый список захвата вы будете изучать далее, однако если интересно можете изучить самостоятельно тему сейчас и использовать слабую ссылку:
```suggestion
        let action = UIAlertAction(title: result.buttonText, style: .default) { [weak self] _ in
```